README.md

```diff
@@ -59,7 +59,7 @@
 - **Community**
     - [📊 System Requirements](#system-requirements-)
     - [🗨️ FAQ (Frequently Asked Questions)](#faq-frequently-asked-questions-)
-    - [📣 More Contributers Wanted](#more-contributers-wanted-)
+    - [📣 More Contributors Wanted](#more-contributers-wanted-)
```

Comment:
```suggestion
    - [📣 More Contributors Wanted](#more-contributors-wanted-)
```

Thanks :heart: for taking the time to put up such a wonderful PR :slightly_smiling_face: , I really appreciate :+1: Keep it up :+1:  but I would like to suggest making this change as well as this change will make the link refer to the appropriate heading.